docs/ml-linalg-guide.md

```diff
@@ -46,8 +46,7 @@ The installation should be done on all nodes of the cluster. Generic version of
 
 For Debian / Ubuntu:
 ```
-sudo apt-get install libopenblas-base
-sudo update-alternatives --config libblas.so.3
+sudo apt-get install libopenblas-dev
```

Comment:
`libopenblas-base` is removed in Debian 12 and Ubuntu 24.04, `libopenblas-dev` should be used instead.

https://github.com/luhenry/netlib/blob/6835050840ead1a724e2f305875c92d7cc21f834/.github/workflows/build-and-test.yml#L31

also `update-alternatives --config libblas.so.3` does not work, and it's variant in different CPU-arch OS.

```
root@0bef5c80cdaa:/# update-alternatives --config libblas.so.3
update-alternatives: error: no alternatives for libblas.so.3
```

Given it already allows using `-Ddev.ludovic.netlib.lapack.nativeLib=...` to choose the native libraries, I would suggest eliminating how to use `alternatives` to manage the OS default library in our docs.